packages/common/src/web3ProofTypes.ts

```diff
@@ -163,6 +165,39 @@ export type VerifiableCredentialProof = {
     ];
 };
 
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types */
+
+/**
+ * Replacer to ensure dates are stringified to the timestamp attribute format.
+ */
+export function replaceDateWithTimeStampAttribute(
```

Comment:
These two functions should perhaps be moved out of the types file and somewhere else.

Reply:
Moved them together with Verifiable Presentation.
+ made tests for them
+ fixed the bug I found due to the tests :upside_down_face:

Reply:
`new Date(Date.parse(value.timestamp));`?